python/Ganga/test/GPI/Bugs/TestSavannah17080.py

```diff
@@ -12,12 +14,36 @@ def test_CondorConfigDefaults(self):
         j.backend= TestSubmitter()
         b = j.splitter.backs[1]
         b.fail = 'submit'
+        j.parallel_submit = False
 
         assert j.status == 'new'
 
-        j.submit(keep_going=False)
-        assert j.status in ['submitted', 'running']
+        with pytest.raises(Exception):
```

Comment:
Can we be a little more specific with the type of exception raised here in a test which is known to fail?

Reply:
yes